dotCMS/src/main/java/com/dotcms/rendering/velocity/servlet/VelocityModeHandler.java

```diff
@@ -100,7 +102,13 @@ protected void processException(final User user, final String name, final ParseE
     public final String eval() {
         try(ByteArrayOutputStream out = new ByteArrayOutputStream(4096)) {
             serve(out);
-            return new String(out.toByteArray());
+
+            if (ContentSecurityPolicyUtil.isConfig()) {
+                final String htmlCode = new String(out.toByteArray());
```

Comment:
you should use a Enconding here

Reply:
UTF-8

Reply:
done https://github.com/dotCMS/core/pull/21337/commits/f13b003205986c038e04ef138375ec610766da14